src/tooling/piral-cli/src/injectors/piral.ts

```diff
@@ -32,11 +32,16 @@ export default class PiralInjector implements KrasInjector {
 
   setOptions() {}
 
-  sendResponse(path: string, target: string, dir: string, url: string): KrasResponse {
+  sendResponse(path: string, target: string, dir: string, url: string, recursionDepth: number = 0): KrasResponse {
```

Comment:
Just `recursionDepth = 0` (the `number` is inferred).

Reply:
adapted; removed the type declaration